etc/ci/report_aggregated_expected_results.py

```diff
@@ -97,28 +97,32 @@ def to_html(self, level: int = 0) -> ElementTree.Element:
 
 def get_results() -> Optional[Item]:
     unexpected = []
-    known_intermittents = []
     for filename in sys.argv[1:]:
         with open(filename, encoding="utf-8") as file:
-            data = json.load(file)
-            unexpected += data["unexpected"]
-            known_intermittents += data["known_intermittents"]
-
+            unexpected += json.load(file)
     unexpected.sort(key=lambda result: result["path"])
-    known_intermittents.sort(key=lambda result: result["path"])
+
+    def is_flaky(result):
+        return result["flaky"]
+
+    def is_stable_and_known(result):
+        return not is_flaky(result) and result["issues"]
+
+    def is_unexpected_stable(result):
+        return not is_flaky(result) and not is_stable_and_known(result)
```

Comment:
the logic behind this wasn’t obvious to me, since it expands to (not flaky and not (not flaky and has issues)), which is (not flaky and (flaky or no issues)), which smells contradictory (despite being correct). i wonder if this would be easier to understand if expressed as

```python
def is_stable_and_unexpected(result):
    return not is_flaky(result) and not result["issues"]
```

Reply:
That makes sense to me. I've done this rename.